README.md

```diff
@@ -257,6 +270,10 @@ In the `proc_grps.json` file, specify your copybook folders in a `libs` array, a
                     "libs": [
                 "LIB3", "LIB4"
             ]
+                }, 
+                {
+                    "name": "DB2",
```

Comment:
same here - it should be: “DB2/DATACOM”

Reply:
@VitGottwald will it make sense to rename the preprocessor into "SQL"?

Reply:
That would make a lot more sense

Reply:
changed in  https://github.com/eclipse-che4z/che-che4z-lsp-for-cobol/pull/1905
It should be `"name": "SQL"` now